src/background/messaging/rpc-methods/open.ts

```diff
@@ -13,18 +13,37 @@ import { trackRpcRequestSuccess } from '../rpc-message-handler';
 
 export async function rpcOpen(message: OpenRequest, port: chrome.runtime.Port) {
   const { urlParams, tabId } = makeSearchParamsWithDefaults(port, [['requestId', message.id]]);
-  const { id } = await triggerRequestWindowOpen(RouteUrls.Home, urlParams);
-  void trackRpcRequestSuccess({ endpoint: message.method });
 
-  listenForPopupClose({
-    tabId,
-    id,
-    response: makeRpcErrorResponse('open', {
+  try {
+    const { id } = await triggerRequestWindowOpen(RouteUrls.Home, urlParams);
+    void trackRpcRequestSuccess({ endpoint: message.method });
+
+    chrome.tabs.sendMessage(
+      tabId,
+      makeRpcSuccessResponse('open', {
+        id: message.id,
+        result: { message: 'Success' },
+      })
+    );
+
+    listenForPopupClose({
+      tabId,
+      id,
+      response: makeRpcErrorResponse('open', {
+        id: message.id,
+        error: {
+          code: RpcErrorCode.USER_REJECTION,
+          message: 'User rejected request to open wallet',
+        },
+      }),
+    });
```

Comment:
This is for the case in which a user closes window during a flow like sending bitcoin.

With an open action, that has no defined goal, closing the window can't be considered a Promise rejection in the same way. I'd remove entirely.